Scripts/Runtime/Entities/TaskDriver/TaskSet/TaskData/JobDataInteraction/EntityPersistentDataReader.cs

```diff
@@ -28,5 +23,26 @@ internal EntityPersistentDataReader(ref UnsafeParallelHashMap<Entity, TData> loo
         {
             get => m_Lookup[entity];
         }
+
+        /// <summary>
+        /// Returns whether the persistent data has no entries.
+        /// </summary>
+        public bool IsEmpty
+        {
+            get => m_Lookup.IsEmpty;
+        }
+
+        /// <summary>
+        /// Returns the number of entries currently stored in persistent data.
+        /// </summary>
+        public int Count
+        {
+            get => m_Lookup.Count();
+        }
```

Comment:
This should be a method instead a property because it's actually going to iterate and calculate the count instead of just returning a cached value.

Reply:
fixed